projects/plugins/jetpack/changelog/blazepress-add-advertising-menu-atomic

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: other
+
+Added Advertising menu entry for Atomic sites gated behind a filter that will be toggled from wpcomsh
```

Comment:
I would recommend adding a category and using the present tense here. Maybe something like this?

```suggestion
WordPress.com toolbar: allow enabling an "Advertising" menu item via a new filter.
```